### PR TITLE
Add support for Postgres

### PR DESCRIPTION
**Pull request by MatejMecka**
Add support for Data Collection with Postgres

### CODE REVIEW

File: helper.py [hunk 1]
```diff
@@ -1,18 +1,36 @@
 from datetime import datetime
+import os
 import postbin
 
+SQLITE3_ENABLED = True if os.environ['SQLITE3_ENABLED'] == "True" else False
```
Comment (Smephite) [inline helper.py:5]: we could use `True if not 'SQLITE3_ENABLED' in os.environ else os.envior['SQLITE3_ENABLED']` to allow env var not being set

File: main.py [hunk 2]
```diff
@@ -17,6 +18,7 @@
 NOTIFY_USER = os.environ['NOTIFY_USER']
 
 IGNORED_CHANNELS = [763798356484161569, 772838189920026635,  839229026194423898] if not 'DISCORD_IGNORED_CHANNELS' in os.environ else json.load(os.environ['DISCORD_IGNORED_CHANNELS'])
+SQLITE3_ENABLED = True if os.environ['SQLITE3_ENABLED'] == "True" else False
```
Comment (Smephite) [inline main.py:21]: same as `helper.py`

File: helper.py (hunk 1 above)
Comment (MatejMecka) [inline helper.py:5]:
```suggestion
SQLITE3_ENABLED = True if not 'SQLITE3_ENABLED' in os.environ else os.envior['SQLITE3_ENABLED']
```

File: main.py (hunk 2 above)
Comment (MatejMecka) [inline main.py:21]:
```suggestion
SQLITE3_ENABLED = True if not 'SQLITE3_ENABLED' in os.environ else os.envior['SQLITE3_ENABLED']
```

**Review by Smephite — APPROVED** [review]
Seems good enough for now!

**Pull request merged**